Rivet/rivet.ps1

```diff
@@ -168,7 +171,12 @@ param(
     [Parameter(ParameterSetName='Checkpoint')]
     [Switch]
     # Checkpoints the current state of the database so that it can be re-created.
-    $Checkpoint
+    $Checkpoint,
+
+    [Parameter(ParameterSetName='InitializeSchema')]
+    [Switch]
+    # Initializes the database using the contents of the schema.ps1 file that was created from checkpoint.

```

Comment:
Rename to `Initializes the database, including baseline schema. Use the `-Checkpoint` switch to create a database baseline.`